### PR TITLE
LPD-31212 Provide an escape hatch to inject XSS in rich text editor

### PR DESCRIPTION
**Pull request by dsanz**
Reference: https://liferay.atlassian.net/browse/LPD-31212

In this PR, we are:

- Creating a new release FF to disable XSS validations in the rich text editor (ddm-form infra)
- Applying it to prevent rich text content from going through the sanitization process
- Adding functional tests on the forms module

### CODE REVIEW

**Comment by dsanz** [comment]
oh, commit messages are also SFed! changing

**Comment by dsanz** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -7405,6 +7405,8 @@ feature.flag.LPD-24498.description=Automatically register an object action to en
 feature.flag.LPD-24498.title=Commerce Order Email Notification Template
 feature.flag.LPD-29516.description=Allows users to upload images to the blog repository even when the workflow is active.
 feature.flag.LPD-29516.title=Image Uploads to Blog Repository with Active Workflow
+feature.flag.LPD-31212.description=Users can create rich text content that includes scripting snippets.
+feature.flag.LPD-31212.title=Disable XSS validations in Rich Text Editor
```
Comment (danvb112) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7409]: Hey @dsanz I think you forgot to run `gw buildlang` to generate the translations, right?

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @dsanz, the code looks good to me, only the buildlang is missing :smile:

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (dsanz) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7409]: yep, I the past I always did that step, nowadays I'm under the impression this thing happens under the hood every in a while.

I've added an extra commit

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:forms

**Comment by marcelabc** [comment]
running CI to anticipate results

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by marcelabc** [comment]
fixed rebase errors and resent here: https://github.com/liferay-objects/liferay-portal/pull/3518

**Pull request closed without merging**